docker-compose.yml

```
@@ -58,6 +58,24 @@ services:
       - WORKERQUEUE=DeadLetterQueue
       - DEV_MODE=0
 
+  rollback-handler:
+    build:
+      context: worker/rollback-handler
+    restart: unless-stopped
+    volumes:
+      - ./geoserver_data:/home/data:Z
```

currently we set the volumes all to the same directory

```suggestion
      - ./geoserver_data:/opt/geoserver_data/:Z
```